generators/add-action/asset-compute/index.js

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+// #########################################################################################
+// 3rd party template
+// #########################################################################################
+
+const path = require('path')
+const ActionGenerator = require('../../../lib/ActionGenerator')
+
+class AssetComputeGenerator extends ActionGenerator {
+  constructor (args, opts) {
+    super(args, opts)
+    this.props = {}
+  }
+
+  async prompting () {
+    this.props.actionName = await this.promptForActionName('example of a custom worker for the Adobe Asset Compute service', 'example')
+    this.props.actionDestPath = this.destinationPath(`worker-${this.props.actionName}.js`)
+  }
+
+  writing () {
+    this.sourceRoot(path.join(__dirname, './templates'))
+
+    this.addAction(this.props.actionName, './_worker.js', {
+      tplContext: this.props,
+      dependencies: {
+        '@adobe/asset-compute-sdk': '^1.0.2'
+      },
+      devDependencies: {
+        '@adobe/aio-cli-plugin-asset-compute': '^1.0.1'
+      },
+      dotenvStub: {
+        label: 'please provide the following environment variables for the Asset Compute devtool',
+        vars: [
+          'AIO_INTEGRATION_FILE_PATH',
+          'ASSET_COMPUTE_URL',
+          'S3_BUCKET',
+          'AWS_ACCESS_KEY_ID',
+          'AWS_SECRET_ACCESS_KEY',
+          'AWS_REGION'
```

Comment:
I would switch to Azure instead of S3 since (I think?) most customers will be using azure:
```
AZURE_STORAGE_ACCOUNT
AZURE_STORAGE_KEY
AZURE_STORAGE_CONTAINER_NAME
```

Reply:
Kept both, clarified label that they must chose one cloud, AWS or Azure.